src/zc/table/batching.py

```diff
@@ -57,20 +57,20 @@ def setPrefix(self, prefix):
         super(Formatter, self).setPrefix(prefix)
         self._batch_start_computed = False
 
-    @apply
```

Comment:
How did this package ever claim Python 3 support?  `apply` does not exist in Python 3!

Does zc.table.batching have no tests at all?

Reply:
Yeah!

```
Name                                                                Stmts   Miss  Cover
---------------------------------------------------------------------------------------
.tox/coverage/lib/python3.7/site-packages/zc/table/__init__.py          2      0   100%
.tox/coverage/lib/python3.7/site-packages/zc/table/batching.py         76     76     0%
```

Zero.